docs/quickstart.md

```diff
@@ -86,3 +86,52 @@ time_series_causal_graph.extend_graph(backward_steps=2)
 # and the following edges
 # X1 lag(n=1) -> X2, X2 lag(n=1) -> X2, X1 lag(n=2) -> X2 lag(n=1), X2 lag(n=2) -> X2 lag(n=1)
 ```
+
+## Time Series Causal Graphs
+
+The `cai_causal_graph.time_series_causal_graph.TimeSeriesCausalGraph` class extends the 
+`cai_causal_graph.causal_graph.CausalGraph` class to deal with time series representations of causal graphs. The main differences with respect to the `cai_causal_graph.causal_graph.CausalGraph` class are:
+- each node is associated with a `variable_name` and a `time_lag` attribute.
+- a new `from_adjacency_matrices` method to create a `TimeSeriesCausalGraph` from a dictionary of adjacency matrices where the keys are the time lags.
+
+`TimeSeriesCausalGraph` is aware of the time lags of the variables and can be extended backwards and forward in time.
+
+If you want to convert a `cai_causal_graph.causal_graph.CausalGraph` to a `cai_causal_graph.time_series_causal_graph.TimeSeriesCausalGraph`, you can use the `cai_causal_graph.time_series_causal_graph.TimeSeriesCausalGraph.from_causal_graph` method as shown below.
+
+```python
+
+from cai_causal_graph import CausalGraph
+from cai_causal_graph import TimeSeriesCausalGraph
+
+# create the causal graph with lagged nodes to show the difference
```

Comment:
This comment is not exactly correct as we don't add lagged nodes here. They are added automatically when the edges are added.